indexer/services/roundtable/src/tasks/take-fast-sync-snapshot.ts

```diff
@@ -0,0 +1,83 @@
+import { logger, stats } from '@dydxprotocol-indexer/base';
+import RDS from 'aws-sdk/clients/rds';
+import { DateTime } from 'luxon';
+
+import config from '../config';
+import {
+  createDBSnapshot,
+  deleteOldFastSyncSnapshots,
+  getMostRecentDBSnapshotIdentifier,
+} from '../helpers/aws';
+
+const statStart: string = `${config.SERVICE_NAME}.fast_sync_export_db_snapshot`;
+
+/**
+ * Checks if the difference between two dates is less than a given interval.
+ *
+ * @param startDate
+ * @param endDate
+ * @param intervalMs
+ */
+function isDifferenceLessThanInterval(
```

Comment:
nit: move this under runTask so that the top level function is at the top of the file

Reply:
done